presto-native-execution/scripts/setup-centos.sh

```diff
@@ -36,6 +36,14 @@ CPU_TARGET="${CPU_TARGET:-avx}"
 SOURCE_FILE="$(dirname "${BASH_SOURCE}")/setup-helper-functions.sh"
 export COMPILER_FLAGS=$(source "$SOURCE_FILE" && echo -n $(get_cxx_flags $CPU_TARGET))
 
+function cmake_install {
```

Comment:
We already have this defined in `setup-helper-functions.sh`. But looks like the file is located in `"$(dirname "${BASH_SOURCE}")/velox/setup-helper-functions.sh"`? I don't see the `COMPILER_FLAGS` being set as well.
@Mionsz can you confirm?

Reply:
yes that is true.

Reply:
Thanks. Can we fix the path and use `cmake_install` from there?

Reply:
Yes, I think it should work. Fixed